maintenance/importConfig.ts

```diff
@@ -34,7 +35,91 @@ import * as fs from "fs-extra";
 import * as JSON5 from "json5";
 import * as path from "path";
 import * as qs from "querystring";
+import { promisify } from "util";
 import xml2json from "xml2json";
+import yargs from "yargs";
+
+const execPromise = promisify(child.exec);
+
+yargs
+	.option("source", {
+		description: "source of the import",
+		alias: "s",
+		type: "array",
+		choices: ["oh", "ozw"], // oh: openhub, ozw: openzwave
+		default: ["oh"],
+	})
+	.option("ids", {
+		description: "devices ids to download",
+		type: "array",
+	})
+	.option("download", {
+		alias: "D",
+		description: "Download devices DB from <source>",
+		type: "boolean",
+		default: false,
+	})
+	.option("clean", {
+		alias: "C",
+		description: "Clean temprorary directory",
+		type: "boolean",
+		default: false,
+	})
+	.option("manufacturers", {
+		alias: "m",
+		description: "Parse and update manufacturers.json",
+		type: "boolean",
+		default: false,
+	})
+	.option("index", {
+		alias: "i",
+		description: "Parse and update devices index.json",
+		type: "boolean",
+		default: false,
+	})
+	.option("devices", {
+		alias: "d",
+		description: "Parse and update devices configurations",
+		type: "boolean",
+		default: false,
+	})
+	.example(
+		"import -s ozw -Dmid",
+		"Download and parse OpenZwave db (manufacturers, index, devices)",
+	)
+	.example(
+		"import -s oh -Dmid",
+		"Download and parse OpenHub db (manufacturers, index, devices)",
+	)
+	.example(
+		"import -s oh -D --ids 1234 5678",
+		"Download OpenHub devices with ids `1234` and `5678`",
+	)
+	.help()
+	.version(false)
+	.alias("h", "help");
+
+class Program {
+	public source: string[];
+	public ids: string[] | undefined;
+	public download: boolean;
+	public clean: boolean;
+	public manufacturers: boolean;
+	public index: boolean;
+	public devices: boolean;
+
+	constructor(json: any) {
```

Comment:
You're losing the yargs type definitions this way.
I think you could just `pick` the wanted properties from `yargs.argv` or use `.argv` directly, like here:
https://github.com/ioBroker/create-adapter/blob/master/src/cli.ts#L59-L62

Reply:
I saw that but in this way it's much better typed IMO, if you prefer that I can remove the Program class and use that

Reply:
lets keep it like this for now. I thought that the types for argv are pretty good, but I haven't tested it in this case.

Reply:
Actually, I'll push a commit where I changed that. The entire class is unnecessary